spec/models/record_spec.rb

```diff
@@ -5,4 +5,48 @@
   it { should validate_presence_of :user }
   it { should validate_presence_of :disposition_date }
   it { should belong_to(:user) }
+
+  let!(:convicted_banned) {
+    FactoryGirl.create(:record, convicted: true, crime_name: "Unlicensed Sale of Ammunition")  }
```

Comment:
Line is too long. [96/80]<br>Expression at 10, 96 should be on its own line.<br>Unnecessary spacing detected.